airflow/utils/log/logging_mixin.py

```diff
@@ -209,9 +209,9 @@ def __init__(self, stream):
     @property
     def stream(self):
         """Returns current stream."""
-        from airflow.settings import IS_K8S_EXECUTOR_POD
+        from airflow.settings import IS_EXECUTOR_CONTAINER, IS_K8S_EXECUTOR_POD
 
-        if IS_K8S_EXECUTOR_POD:
+        if IS_K8S_EXECUTOR_POD or IS_EXECUTOR_CONTAINER:
```

Comment:
This is core change.

two issues:
1. Is it possible to extract the core changes to separate PR and merge them before this one? (will make release manager life easier)
2. can you clarify how the executor going to work for users who have Airflow version that does not include these core changes? (The question here is if this provider needs min Airflow version that include these core changes)

Reply:
The only issue you'd have is that task logs won't be redirected so the user wont see the logs they expect. We should document this behaviour, but I'm not sure we need to set the min version? Sine as long as users are okay with not seeing task logs they can use older airflow versions. But I'm interested to hear other people's perspectives on this one.

CC @shubham22

Reply:
And what about seperating the PR core and providers. Can we do it?

Reply:
I'll work on that change @eladkal

Reply:
This is done and merged :+1: